src/wp-includes/interactivity-api.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Interactivity API: Functions and hooks
+ *
+ * @package WordPress
+ * @subpackage Interactivity API
+ */
+
+/**
+ * Registers the interactivity modules.
+ */
+function wp_interactivity_register_script_modules() {
+	wp_register_script_module(
+		'@wordpress/interactivity',
+		includes_url( '/js/dist/interactivity.min.js' ),
+		array()
+	);
+
+	wp_register_script_module(
+		'@wordpress/interactivity-router',
+		includes_url( '/js/dist/interactivity-router.min.js' ),
+		array( '@wordpress/interactivity' ),
+	);
+}
+
+add_action( 'init', 'wp_interactivity_register_script_modules' );
```

Comment:
Should we introduce a new action similar to `wp_default_scripts`?

Inside `src/wp-includes/default-filters.php`, we would put:
```php
add_action( 'wp_default_script_modules', 'wp_default_script_modules_packages' );
```

Reply:
That makes sense, I believe this is meant as a start though. @luisherranz is planning a follow-up for this PR to bring the full interactivity API.

Reply:
Yes. Once this is committed, my plan is to delete it in https://github.com/WordPress/wordpress-develop/pull/5953, which already includes these filters.

Reply:
In #5953, it is no longer part of the `init` hook, but instead the script modules for Interactivity API get registered when someone calls `wp_interactivity()` function.